Alignment/CocoaFit/src/CocoaDBMgr.cc

```diff
@@ -98,60 +88,53 @@ bool CocoaDBMgr::DumpCocoaResults() {
 
   if (gomgr->GlobalOptions()["writeDBAlign"] > 0) {
     // Build DT alignments and errors
-    std::pair<Alignments*, AlignmentErrorsExtended*> dtali = BuildAlignments(true);
-    Alignments* dt_Alignments = dtali.first;
-    AlignmentErrorsExtended* dt_AlignmentErrors = dtali.second;
+    const auto& dtali = BuildAlignments(true);
```

Comment:
Maybe i'm misunderstanding C++11 here...  But I'm not sure that this will work. BuildAlignments is returning by value a std::pair<std::unique_ptr<Alignments>, std::unique_ptr<AlignmentErrorsExtended> >.   No object declared in the stack is holding the return value - so maybe it is presumably going away? If this is the case, the reference (const auto& dtali ) will then point to invalid memory?

Reply:
If I understood @ggovi correctly from private conversation, this particular change is not a concern.
@tvami please let me know, if you really prefer to have `BuildAlignments` to return objects on the stack.

Reply:
Hi @mmusich I trust your judgment, if you think we are fine, I can sign. Should we ask the alignment team to double check this in the IB?

Reply:
> Should we ask the alignment team to double check this in the IB?

Yes, that would be good.

Reply:
@vbotta @antoniovagnerini please have a look at this when the IB is out, thanks!